.changeset/common-toes-dream.md

```diff
@@ -0,0 +1,7 @@
+---
+"@liam-hq/db-structure": patch
+"@liam-hq/erd-core": patch
```

Comment:
The changes in this package are not critical.
Should we remove this line from the changeset?

Reply:
This time, because of the user impact, I thought it was a good idea to keep it that way 👍🏻